src/windows/BarcodeScannerProxy.js

```diff
@@ -17,95 +83,105 @@ module.exports = {
      * @param  {array} args       Arguments array
      */
     scan: function (success, fail, args) {
-
         var capturePreview,
             capturePreviewAlignmentMark,
             captureCancelButton,
             capture,
             reader;
-        
+
+        function updatePreviewForRotation() {
+            if (!capture) {
+                return;
+            }
+            var videoEncodingProperties = capture.videoDeviceController.getMediaStreamProperties(Windows.Media.Capture.MediaStreamType.VideoPreview);        
+
+            var rotDegree = videoPreviewRotationLookup(Windows.Graphics.Display.DisplayInformation.getForCurrentView().currentOrientation);
```

Comment:
is equal to `var rotDegree = videoPreviewRotationLookup(event.target);` - see previous comment